<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KPasswordDialog</class>
 <widget class="QDialog" name="KPasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>194</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="vBoxLayout">
   <item>
    <layout class="QHBoxLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="prompt">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>10</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Supply a username and password below.</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>400</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="KTitleWidget" name="errorMessage"/>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="userNameLabel">
       <property name="text">
        <string>Username:</string>
       </property>
       <property name="buddy">
        <cstring>userEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="userEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="anonymousCheckBox">
       <property name="text">
        <string>Anonymous</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="domainLabel">
       <property name="text">
        <string>Domain:</string>
       </property>
       <property name="buddy">
        <cstring>domainEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="domainEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="passwordLabel">
       <property name="text">
        <string>Password:</string>
       </property>
       <property name="buddy">
        <cstring>passEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="passEdit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="keepCheckBox">
       <property name="text">
        <string>Remember password</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KTitleWidget</class>
   <extends>QWidget</extends>
   <header>ktitlewidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>userEdit</tabstop>
  <tabstop>anonymousCheckBox</tabstop>
  <tabstop>domainEdit</tabstop>
  <tabstop>passEdit</tabstop>
  <tabstop>keepCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>KPasswordDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>KPasswordDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
